src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/command/rulemanager/OfBatchHolder.java

```diff
@@ -107,6 +110,16 @@ public void otherFail(String message, Throwable throwable) {
         this.failCause = throwable;
     }
 
+    /**
+     * Process other fail during command processing and record failed commands.
+     */
+    public void otherFail(String message, Throwable throwable, Collection<UUID> failedCommands) {
```

Comment:
I don't think it's a good idea to have overloaded methods which, in fact, does two things. Should be renamed to something like `otherFailAndRecord()`.